srcpkgs/folks/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'folks'
 pkgname=folks
 version=0.14.0
```

Comment:
0.15 is out (not sure if odds are dev releases)

Reply:
afaik folks follows the gnome release style. 0.15 is unstable.